.golangci.yml

```diff
@@ -0,0 +1,40 @@
+linters:
+  enable:
+    - gofmt
+    - govet
+    - errcheck
+    - staticcheck
+    - gosimple
+    - ineffassign
+    - typecheck
+    - unused
+    - misspell
+    - gocyclo
+    - gosec
+    - revive
```

Comment:
My recommendation is to enable all by default, and manually disable those that you think are irrelevant. Here I see a lot of linters missing from this list that I consider very useful, so it would be simpler to just enable all